sample_command_policy.json

```diff
@@ -0,0 +1,28 @@
+{
+  "commands" : [
+    [ 
+        "cd examples/c_code_proj;  ./makefile",
+        "ls -ltr examples/c_code_proj/test",
+        "build_snap",
+        "default_policy.json"
+    ],
+    [
+        "cd examples/java_maven_proj;  mvn clean",
+        "ls -ltr examples/java_maven_proj/target",
+        "build_main",
+        "README.md"
+    ],
+    [
+        "cd tuf; sudo python setup.py install",
+        null,
+        "build_tuf_install",
+        null
+    ],
+    [
+        "cd examples/c_code_proj;  ./makefile_err",
+        "ls -ltr testErr",
+        "build_snapErr",
+        null
+    ]
+  ]
+}
```

Comment:
I want this file to be clear and simple.  The guys setting this up have to be really crisp about what they're doing.  

I'm planning to update our wiki once this is pulled in with the different options.  It's sort of straightfwd to:
[0] - command to execute (COMMAND option on commandline)
[1] - verify command that the dev wants to run to check this execution (ex:  ls test for the c binary)
[2] - command name description (the metadata files will also be prepended with this value, formatted <name>_metadata.json
[3] - input file (input option on commandline)